code/datums/status_effects/debuffs.dm

```diff
@@ -165,10 +165,9 @@
 		owner.adjustStaminaLoss(healing)
 	if(human_owner && human_owner.drunkenness)
 		human_owner.drunkenness *= 0.997 //reduce drunkenness by 0.3% per tick, 6% per 2 seconds
-	if(prob(20))
-		if(carbon_owner)
-			carbon_owner.handle_dreams()
-		if(prob(10) && owner.health > owner.crit_threshold)
+	if(prob(20) && carbon_owner?.client)
+		carbon_owner.handle_dreams()
+		if(prob(10) && carbon_owner.client && owner.health > owner.crit_threshold)
```

Comment:
```suggestion
		if(prob(10) && owner.health > owner.crit_threshold)
```
shouldn't be necessary bc this is inside a check that already checks for the client